src/main/java/com/sourcegraph/cody/chat/CodeEditorFactory.java

```diff
@@ -179,9 +171,8 @@ private ActionListener copyCodeListener(EditorEx editor, JButton copyButton) {
       timer.start();
 
       lastCopiedText = text;
-      ApplicationManager.getApplication()
```

Comment:
No need to run it on separate thread as telemetry calls just runs an agent call which is done on a separate thread anyway.